contrib/win32/win32compat/shell-host.c

```diff
@@ -980,6 +1025,20 @@ ConsoleEventProc(HWINEVENTHOOK hWinEventHook,
 }
 
 DWORD 
+GetCurrentTerminalType(void)
+{
```

Comment:
windows cmd.exe/powershell doesn't have the environment variable "TERM" set.. so if we take a dependency on the "TERM" then you need to modify the ssh client code to send the "TERM" as "xterm" when the "TERM" environment variable is not found.. fyi, ssh client will send the "TERM" to the ssh server during the session establishment..